src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/Strings.resx

```diff
@@ -201,6 +201,9 @@
     <value>Workload version {0}, which was specified in {1}, was not found. Run "dotnet workload restore" to install this workload version.</value>
     <comment>{Locked="dotnet workload restore"}</comment>
   </data>
+  <data name="ShouldInstallAWorkloadSet" xml:space="preserve">
+    <value>Workloads are configured to install and update using workload versions, but none were found. Run "dotnet workload restore" to install a workload version.</value>
```

Comment:
I thought we discussed moving that message to the bottom of the output from workload --info? Though perhaps I misremembered.

Workload sets are basically workload versions, but it was decided that calling them sets was confusing, so workload version is the customer-facing phrasing.

Reply:
I see. But the toggle switch (maybe internal only) uses the workload-set terminology. `dotnet workload config --update-mode workload-set` 😭

Reply:
Workload sets and workload versions are the same thing.  I try to use "workload versions" as the user-facing term because it can be decently understood without learning a new concept, but that didn't really work with the config setting so they both show up.

I've added an extra blank line.